src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,39 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+        int supplyCounter = 0;
+        int buyCounter = 0;
```

Comment:
```suggestion
        String dataFromFile = readData(fromFileName);
        String report = createReport(dataFromFile);
        writeToFile(report, toFileName);
```
lets divide the logic into 3 separate methods